modules/exploits/linux/http/apache_superset_cookie_sig_rce.rb

```diff
@@ -273,6 +273,7 @@ def mount_internal_database
     )
 
     fail_with(Failure::Unreachable, "#{peer} - Could not connect to web service - no response") if res.nil?
+    fail_with(Failure::UnexpectedReply, "#{peer} - Failed to mount the internal database: #{datastore['DATABASE']}") if res.code == 422
```

Comment:
I dont think I tested for this. Good to know it gives a super easy HTTP code to distinguish from!